lib/rucio/web/ui/static/subscriptions.js

```diff
@@ -135,7 +135,7 @@ $(document).ready(function(){
         insertParam('account', chosen_account);
     }
 
-    var link_list_rules = '/list_rules?account=' + chosen_account;
+    var link_list_rules = '/r2d2?account=' + chosen_account;
```

Comment:
unrelated change?

Reply:
Not really. In the web.py ui, list_rules is just a redirect to r2d2. This saves me the endpoint in the flask app and it won't break web.py.